### PR TITLE
joinPracticeLobbyCoachTeam method added

### PR DESCRIPTION
**Pull request by herenickname**

### CODE REVIEW

File: handlers/lobbies.js [hunk 1]
```diff
@@ -438,6 +438,31 @@ Dota2.Dota2Client.prototype.joinPracticeLobbyTeam = function(slot, team, callbac
...
+ */
+Dota2.Dota2Client.prototype.joinPracticeLobbyCoachTeam = function(team, callback) {
+    callback = callback || null;
+    if (typeof team === 'undefined') team = Dota2.schema.lookupEnum("DOTA_GC_TEAM").values.DOTA_GC_TEAM_GOOD_GUYS;
+    
+    var _self = this;
+
+    this.Logger.debug("Sending match CMsgPracticeLobbySetCoach request");
+    
+    var payload = {
```
Comment (Crazy-Duck) [inline handlers/lobbies.js]: Should be
```
var payload = new Dota2.schema.CMsgPracticeLobbySetCoach({
    "team": team
});
```

File: handlers/lobbies.js [hunk 2]
```diff
@@ -438,6 +438,31 @@ Dota2.Dota2Client.prototype.joinPracticeLobbyTeam = function(slot, team, callbac
...
+        "team": team
+    };
+    this.sendToGC(  Dota2.schema.lookupEnum("EDOTAGCMsg").values.k_EMsgGCPracticeLobbySetCoach,
```
Comment (Crazy-Duck) [inline handlers/lobbies.js]: Should be 
```
this.sendToGC(Dota2.schema.EDOTAGCMsg.k_EMsgGCPracticeLobbySetCoach, 
                        payload, 
                        onPracticeLobbyResponse,
                        callback
);
```

Comment (herenickname) [reply handlers/lobbies.js]: updated

File: handlers/lobbies.js (hunk 1 above)
Comment (herenickname) [reply handlers/lobbies.js]: updated

**Pull request closed without merging**